src/pages/Signup/index.jsx

```diff
@@ -112,16 +113,22 @@ const SignupScreen = () => {
     }
 
     if (submitable) {
-      const usernameDoc = db.doc(`usernames/${username}`);
+      const usernameDoc = db.collection(`users`);
```

Comment:
we also don't need this variable anymore, we can directly call it in the function below

Reply:
Yup, Done!